pkg/agent/as3/as3Manager.go

```diff
@@ -213,6 +213,7 @@ func updateTenantMap(tempAS3Config AS3Config) AS3Config {
 }
 
 func (am *AS3Manager) postAS3Declaration(rsReq ResourceRequest) (bool, string) {
+	defer log.Timeit("info")("")
```

Comment:
Please change info to debug here as well. Rest all look good.

Reply:
Done